phys/module_diag_hailcast.F

```diff
@@ -1375,14 +1439,21 @@ SUBROUTINE HEATBUD(TS,TSm1,TSm2,FW,TC,VT,DELRW,D,DENSA,GM1,GM,DGM,DGMW,     &
          
          IF (TCC.LT.0.) THEN
             !Original Hailcast algorithm
-            FW=FW-FW*DGM/GM1+SEKDEL/(GM1*ALF)*               &
-                (2.*PI*D*(AH*AK*TCC-AE*ALV*DI*DELRWC)+          &
-                DGMW/SEKDEL*(ALF+CW*TCC)+DGMI/SEKDEL*CI*TCC)
+            !FW=FW-FW*DGM/GM1+SEKDEL/(GM1*ALF)*               &
```

Comment:
Remove old code

Reply:
This section is important to keep in, as it shows readers what the original HAILCAST algorithm had. Some studies in the literature still use this original algorithm.